package.json

```diff
@@ -11,11 +11,12 @@
     "codeceptjs": "^3.0.4",
     "codeceptjs-resemblehelper": "^1.9.3",
     "deep-equal-in-any-order": "^1.0.28",
+    "html_codesniffer": "^2.5.1",
     "jwt-decode": "^3.1.2",
     "mocha-junit-reporter": "^2.0.0",
     "mocha-multi": "^1.1.3",
     "mochawesome": "^6.2.1",
-    "nightmare": "^3.0.2",
+    "nightmare": "^2.8.0",
```

Comment:
Is this version change intended and required for accessibility tests?

Reply:
I had not made this change not sure how it got changed. However, I have updated nightmare back to ^3.0.2